src/main/java/com/intel/databackend/api/kafka/KafkaSenderService.java

```diff
@@ -57,17 +61,24 @@ public void createTopic() {
         if (kafkaProducer != null) {
             ZkClient zkClient = null;
             String brokerURI = null;
+	    ZkUtils zkUtils = null;
             try {
                 topic = serviceConfigProvider.getKafkaTopicName();
                 Integer partitions = serviceConfigProvider.getKafkaPartitionsFactor();
                 Integer replicationFactor = serviceConfigProvider.getKafkaReplicationFactor();
                 Integer timeoutInMs = serviceConfigProvider.getKafkaTimeoutInMs();
                 brokerURI = serviceConfigProvider.getZookeeperUri();
                 zkClient = new ZkClient(brokerURI, timeoutInMs, timeoutInMs, ZKStringSerializer$.MODULE$);
+		// Security for Kafka was added in Kafka 0.9.0.0
+       		boolean isSecureKafkaCluster = false;
```

Comment:
@jelyoussefi What will this property control? What does it mean to have a secure Kafka cluster?